interface/main/tabs/main.php

```diff
@@ -361,19 +369,44 @@ function isEncounterLocked( encounterId ) {
 $('#updater-options').click(function () {
     $('#updater-settings-iframe').iziModal('open');
 });
-
+var timer;
+var secondtimer;
 $('#updater-icon').hover(function () {
     $(this).css("border", "2px solid #000");
     $('#developer-mode').fadeIn(100);
     $('#updater-options').fadeIn(100);
+}, function(){
+    timer = setTimeout(function(){
+        $('#developer-mode').fadeOut(100);
+        $('#updater-options').fadeOut(100);
+        $('#updater-icon').css("border", "none");
+    }, 3000);
+});
+$('#updater-icon-wrapper').hover(function(){}, function(){
+    secondTimer = setTimeout(function(){
+        $('#developer-mode').fadeOut(100);
+        $('#updater-options').fadeOut(100);
+        $('#updater-icon').css("border", "none");
+    }, 100);
+})
+$('#developer-mode').hover(function () {
+    clearTimeout(timer);
+    clearTimeout(secondTimer);
+});
+$('#updater-options').hover(function () {
+    clearTimeout(timer);
+    clearTimeout(secondTimer);
 });
+
 $('#updater-icon').click(function () {
     $('#updater-iframe').iziModal('open');
 });
 $('body').hover(function () {
     $('#developer-mode').css("display", "none");
     $('#updater-options').css("display", "none");
     $(this).css("border", "0px solid #000");
+}, function(){
+
```

Comment:
What this for i calls no method... nor does works

Reply:
@muarachmann It's a mistake in my code, I'll remove it. I was meant to put it on another hover function but probably put it here instead. It doesn't do anything though so it won't break the code

Reply:
@muarachmann removed it!